web/core/src/jsMain/kotlin/org/jetbrains/compose/web/css/selectors/CSSSelectors.kt

```diff
@@ -23,13 +23,18 @@ sealed class Nth {
 
 open class CSSSelector {
     override fun equals(other: Any?): Boolean {
-        return this === other || toString() == other.toString()
+        return this === other || asString() == (other as? CSSSelector)?.asString()
     }
```

Comment:
This is irrelevant to the PR itself but I want to use this as a chance to refresh a bit the context of having this equals at all - do we anywhere in code rely on its existence?

Reply:
It's because i had plan to make our css reactive - so we can compare selectors equality. At least.